site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -56,16 +54,16 @@ models). You can define a flavor using a [ResourceFlavor object](/docs/concepts/
 In a process called [admission](/docs/concepts#admission), Kueue assigns to the
 [Workload pod sets](/docs/concepts/workload#pod-sets) a flavor for each resource the pod set
 requests.
-Kueue assigns the first flavor in the ClusterQueue's `.spec.resources[*].flavors`
-list that has enough unused `min` quota in the ClusterQueue or the
+Kueue assigns the first flavor in the ClusterQueue's `.spec.resourceGroup.flavors`
+list that has enough unused `nominalQuota` quota in the ClusterQueue or the
 ClusterQueue's [cohort](#cohort).
 
-### Codependent resources
+### Resource Groups
 
 It is possible that multiple resources in a ClusterQueue have the same flavors.
 This is typical for `cpu` and `memory`, where the flavors are generally tied to
 a machine family or VM availability policies. When two or more resources in a
```

Comment:
To tie two or more resources to the same set of flavors, you can list them in the same resource group.

Reply:
Dome.